### PR TITLE
reverse proxy support

### PR DESCRIPTION
**Pull request by raphapr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +65,12 @@ Enables secondary MongoDB database configuration.
 ```
 
 Enable and installs pritunl plugins
+```yaml
+    pritunl_reverse_proxy: True
+    pritunl_server_port: 3000
```
Comment (mtulio) [inline README.md:70]: @raphapr it’s very import to define default values for roles variables in **defaults/main.yml** .
I did not see these reverse proxies vars there.

File: tasks/config.yml [hunk 2]
```diff
@@ -51,3 +51,21 @@
...
+  when: pritunl_server_port is defined
+  tags:
+    - pritunl-reverse-proxy
+
+- name: "Set app.server_port to {{ pritunl_server_port }}"
+  shell: "pritunl set app.server_port {{ pritunl_server_port }}"
+  register: resp
+  failed_when: resp.rc != 0
+  changed_when: "'app.server_port = {{ pritunl_server_port }}' in resp.stdout"
+  when: "'app.server_port = {{ pritunl_server_port }}' not in app_server_port.stdout and pritunl_server_port is defined"
```
Comment (mtulio) [inline tasks/config.yml:69]: In the case that you have enabled the rev proxy and wants to change the port, will this condition return false and not change to new one, right? Is that the behavior expected ?

Comment (mtulio) [reply tasks/config.yml:69]: I reviewed the logic. **not in stdout**. :)
This CLI could return an dict to be easy to verify. =p

File: README.md [hunk 3]
```diff
@@ -79,6 +85,7 @@ Example Playbook
     pritunl_secondary_mongodb: True
     pritunl_mongodb_uri: mongodb://db01:27017/pritunl
     pritunl_secondary_mongodb_uri: mongodb://db02:27017/pritunl
+    pritunl_server_port: 3000
```
Comment (mtulio) [inline README.md]: **pritunl_reverse_proxy: True**

**Review by mtulio — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (raphapr) [reply README.md:70]: You're right. Done.

File: tasks/config.yml (hunk 2 above)
Comment (raphapr) [reply tasks/config.yml:69]: Indeed

**Pull request merged**